frontend/src/components/samples/SampleEditContent.js

```diff
@@ -158,14 +155,30 @@ const SampleEditContent = ({ token, samplesByID, sampleKinds, add, update, listT
     onSearchContainer(newData.container, { exact_match: true })
     onSearchCoordinate(newData.coordinate, { exact_match: true })
     onSearchSampleKind(newData.sample_kind)
+    if (!isAdding)
```

Comment:
I'm not sure why it makes a difference whether we are adding a new sample or editing an existing sample. Couldn't we call `form.setFieldsValue` either way? `newData` will have whatever fields it got from EMPTY_SAMPLE. Does that cause an error?